docs/src/_getting-started/part1.md

```diff
@@ -0,0 +1,217 @@
+                start_date: "2021-10-01T00:00:00Z"
+  ```
+
+  Sensitive configuration (like `private_token`) will instead be stored in your project's [`.env` file](/concepts/project#env) so that it will not be checked into version control.
+
+3. Double check the config by running [`meltano config tap-gitlab`](/reference/command-line-interface#config):
+
+   ```bash
+   meltano config tap-gitlab
+   ```
+![Meltano Version](images/part1/gif_meltano_config_show.gif)
+
+
+### Select Entities and Attributes to Extract
+
+Now that the extractor has been configured, it'll know where and how to find your data, but won't yet know which specific entities and attributes (tables and columns) you're interested in.
+
+By default, Meltano will instruct extractors to extract all supported entities and attributes, but we're going to [select specific ones for this tutorial](/guide/integration#selecting-entities-and-attributes-for-extraction). 
+
+1. Find out what entities and attributes are available, using [`meltano select --list --all`](/reference/command-line-interface#select):
+
+   ```bash
+   meltano select tap-gitlab --list --all
+   ```
+  ![Meltano Version](images/part1/gif_meltano_select_all.gif)
+
+
+1. select the entities and attributes for extraction using [`meltano select`](/reference/command-line-interface#select):
+
+   ```bash
+   meltano select tap-gitlab commits id
+   meltano select tap-gitlab commits project_id
+   meltano select tap-gitlab commits created_at
+   meltano select tap-gitlab commits author_name
+   meltano select tap-gitlab commits message
+
+   # Exclude matching attributes of all entities
+   meltano select tap-gitlab --exclude "*" "*_url"
+   ```
+
+   As you can see in the example, entity and attribute identifiers can contain wildcards (`*`) to match multiple entities or attributes at once.
+
+   This will add the [selection rules](/concepts/plugins#select-extra) to your [`meltano.yml` project file](/concepts/project#plugin-configuration):
+
+   ```yml
+   plugins:
+     extractors:
+       - name: tap-gitlab
+         variant: meltanolabs
+         pip_url: git+https://github.com/MeltanoLabs/tap-gitlab.git
+   environments:
+     - name: dev
+       config:
+         plugins:
+           extractors:
+             - name: tap-gitlab
+               config:
+                 projects: meltano/meltano meltano/tap-gitlab
+                 start_date: "2022-03-01T00:00:00Z"
+               select:
+                 - commits.id
+                 - commits.project_id
+                 - commits.created_at
+                 - commits.author_name
+                 - commits.message
+                 - tags.*
+                 - "!*.*_url"
+   ```
+
+   Note that exclusion takes precedence over inclusion, always.
+
+1. Run [`meltano select --list`](/reference/command-line-interface#select) to double-check your selection:
+
+   ```bash
+   meltano select tap-gitlab --list
+   ```
+
+## Add a dummy loader to dump the data into JSON
```

Comment:
Instead of a dummy loader, I think there's (2) simpler paths:

* run `meltano invoke tap-gitlab` and see if data is output on the command line
* run `meltano config tap-gitlab test`. That may not be supported if it's not an SDK-based tap. https://docs.meltano.com/reference/command-line-interface#how-to-use-1

I like not introducing a loader at this stage because it gets folks thinking about testing their extractor in isolation. What do you think?

Reply:
I do see that it would add an extra step between getting them to a `meltano run` command, so perhaps that's an argument against.

Reply:
Mh, I agree the loader is a (light) additional weight. Ideally I would like taps to dump to file with some optional flag, without me needing to add this. Or just printing the data straight to stdout (just the data, nothing else). I added this to just get the data, and nothing else, the data as it will flow into the next step. 

Not sure either here.